Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -352,6 +352,12 @@ protected function initialize()
             }
         }
 
+        foreach ($this->settings['reflection']['ignoredNamespaces'] as $namespace => $ignoreFlag) {
```

Comment:
There are already tests for the `isTagIgnored` method, so they can probably easily be extended/adjusted with namespace cases.

I'd assume Flow itself only cares/knows about ignoring single tags and `isTagIgnored` also checking against a namespace blacklist being an implementation detail, so I'd be in favour of keeping that in this single method. Otherwise we'd just move this complexity one level up in abstraction, which I see no real reason for.